assets/css/main.css

```
@@ -54,6 +54,14 @@ main {
     width: 100%;
 }
 
+.dropzone .dz-preview.dz-image-preview {
+    background: #E1E1E1 !important;
```

how about using this instead?

```suggestion
    background: #transparent !important;
```